worldengine/cli/main.py

```diff
@@ -441,10 +450,10 @@ def main():
     if args.temps and len(args.temps.split('/')) is not 6:
         usage(error="temps must have exactly 6 values")
 
-    temps =[.874, .765, .594, .439, .366, .124]
+    temps = [.874, .765, .594, .439, .366, .124]
     if args.temps:
         temps = args.temps.split('/')
-        for x in range(0,6):
+        for x in range(0,len(temps)):
```

Comment:
should we verify the input is correct? (correct number of items, values in correct range, decreasing order)